plugins/home/README.md

```diff
@@ -0,0 +1,13 @@
+# home
+
+Welcome to the home plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Getting started
+
+Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn start` in the root directory, and then navigating to [/](http://localhost:3000/).
```

Comment:
If the `api-report.md` is important enough, might make sense to link it off this main page.  Obvs this is just boilerplate at this point, so take with a grain of salt given this as a "PRFC" 😉

Reply:
Haha yea, included in dc7c9eb!